cmd/push.go

```diff
@@ -36,37 +36,40 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type PushOptions struct {
```

Comment:
Does this need to be exported? if it does then it needs a comment. It may be worth a comment regardless but it especially needs one when exporting.